extraConfigSriov.py

```diff
@@ -38,6 +40,33 @@ def _sno_make_deploy(
     client.oc("create namespace openshift-sriov-network-operator")
     client.oc("label ns --overwrite openshift-sriov-network-operator " "pod-security.kubernetes.io/enforce=privileged " "pod-security.kubernetes.io/enforce-version=v1.24 " "security.openshift.io/scc.podSecurityLabelSync=false")
 
+    ocp_version = None
+    v, number_of_subs_made = re.subn("^(4[.][0-9]+).*$", "\\1", version)
+    if number_of_subs_made == 1:
+        if v in ("4.14", "4.15", "4.16", "4.17"):
+            # Only these tags exist currently for the default_base_images
+            ocp_version = v
+    if ocp_version is None:
+        logger.error_and_exit(f"unsupported version \"{version}\"")
+
+    # The images at quay.io/thaller were pushed using the script mentioned in
+    # https://github.com/bn222/cluster-deployment-automation/pull/194. You can
+    # use the script, to build your own images and push to your own container
+    # registry.
+    default_base_images = {
+        "SRIOV_INFINIBAND_CNI_IMAGE": f"quay.io/thaller/ib-sriov-cni:{ocp_version}",
+        "SRIOV_CNI_IMAGE": f"quay.io/thaller/sriov-cni:{ocp_version}",
+        "OVS_CNI_IMAGE": "quay.io/kubevirt/ovs-cni-plugin:latest",
+        "SRIOV_DEVICE_PLUGIN_IMAGE": f"quay.io/thaller/sriov-network-device-plugin:{ocp_version}",
+        "NETWORK_RESOURCES_INJECTOR_IMAGE": f"quay.io/thaller/sriov-dp-admission-controller:{ocp_version}",
+        "SRIOV_NETWORK_CONFIG_DAEMON_IMAGE": f"quay.io/thaller/sriov-network-config-daemon:{ocp_version}",
+        "SRIOV_NETWORK_WEBHOOK_IMAGE": f"quay.io/thaller/sriov-network-webhook:{ocp_version}",
+        "SRIOV_NETWORK_OPERATOR_IMAGE": f"quay.io/thaller/sriov-network-operator:{ocp_version}",
+    }
+    # Images can be specified via environment variables, before falling back to
+    # the default.
+    deploy_env.update({k: os.environ.get(k, v) for k, v in default_base_images.items()})
```

Comment:
Can we extract a function and return the default_base_images from the function?

Then we can write the `deploy_env` like this:
`deploy_env.update({k: os.environ.get(k, v) for k, v in default_sriov_quay_images().items()})`

If you have better ideas for the function name, I'm all for it.